tests/app/dao/test_organisation_dao.py

```diff
@@ -356,3 +357,22 @@ def test_get_organisation_by_email_address_ignores_gsi_gov_uk(notify_db_session)
 
     found_org = dao_get_organisation_by_email_address('test_gsi_address@example.gsi.gov.uk')
     assert org == found_org
+
+
+def test_dao_get_organisation_live_services(notify_db_session):
+    org = create_organisation()
+    live_service = create_service(service_name='live service', restricted=False, count_as_live=True)
+    do_not_count_as_live_service = create_service(
+        service_name='do_not_count_as_live_service', restricted=False, count_as_live=False
+    )
+    archived_service_but_count_as_live = create_service(
```

Comment:
I think we should move all testing of logic about which services are live into this test (test_fetch_usage_year_for_organisation_does_not_return_data_for_service_where_count_as_live_is_false) and then assert in `test_fetch_usage_year_for_organisation_only_returns_data_for_live_services` that we are calling this method.

Then this test can test the following cases
- live service
- archived service
- trial service
- live service not to be counted as live because it's one for our testing
- any other types?

and then it's very clear and in one place the behaviour we expect